src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/AuthProviderSpec.scala

```diff
@@ -218,10 +218,6 @@ class AuthProviderSpec extends FreeSpec with ScalatestRouteTest with Matchers wi
 
       // check that the cluster does not exist
       gdDAO.clusters should not contain key (name1)
-
-      //verify we correctly notified the auth provider
-      verify(spyProvider).notifyClusterCreated(userInfo.userEmail, project, name1)
-      verify(spyProvider).notifyClusterDeleted(userInfo.userEmail, project, name1)
```

Comment:
you can do `verify(spyProvider, Mockito.never).blah()` if you want to verify they never got called